frontend/packages/shared/src/utils/featureToggleUtils.ts

```diff
@@ -12,7 +12,8 @@ export type SupportedFeatureFlags =
   | 'exportForm'
   | 'addComponentModal'
   | 'subform'
-  | 'summary2';
+  | 'summary2'
+  | 'codeListEditor';
```

Comment:
We currently don't have such an editor o.O it will likely merge with the functionality Erling is making and then have this as it's common name. It can be named `predefinedCodeListEditor` if that is better?

Reply:
But did we not agree that when using the codelists in the ux-editor --> referring to them with optionsId, we should use the term optionList? 🫣

Reply:
True, i'll update that